BinaryEncoding.md

```diff
@@ -24,6 +24,11 @@ experimented with by application-level decompressing to the layer below. As
 compression techniques  stabilize, they can be standardized and moved into native 
 implementations.
 
+See
+[potential layer 1 compression](https://github.com/WebAssembly/decompressor-prototype/blob/master/CompressionLayer1.md)
+for a proposed future solution for layer 1 structural compression.
```

Comment:
lgtm.

But maybe `solution => approach`, or shorten to `a proposal for layer 1 structural compression`? Just that this isn't a problem/solution type situation.